source/iofs/iofs.go

```diff
@@ -0,0 +1,166 @@
+// +build go1.16
+
+package iofs
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/fs"
+	"path"
+	"strconv"
+
+	"github.com/golang-migrate/migrate/v4/source"
+)
+
+// Driver is a source driver that wraps io/fs#FS.
+type Driver struct {
```

Comment:
Don't export this. e.g. rename to `driver`

Reply:
If did not export it, you will not be able to embed it in the file driver.
httpfs.PartialDriver is public now.

Reply:
I understand it, I will separate into `PartialDriver` and `driver`.